packages/plugin-conversation/test/integration/spec/verbs.js

```diff
@@ -261,6 +262,72 @@ describe(`plugin-conversation`, function() {
           assert.lengthOf(activity.object.mentions.items, 1);
           assert.equal(activity.object.mentions.items[0].id, mccoy.id);
         }));
+
+      skipInNode(describe)(`when there is html tags in rich messages`, () => {
```

Comment:
"when there are"

Also, please add a comment along the lines of "disabled until helper-html has node support"